CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Change log for Microsoft365DSC
 
 # UNRELEASED
+* AADConditionalAccessPolicy
+  * Fixed [#2974](https://github.com/microsoft/Microsoft365DSC/issues/2974)
```

Comment:
Can you please also add a short description of what you fixed in addition to linking to the issue?